app-backend/lambda-overviews/src/main/scala/com/rasterfoundry/lambda/overviews/OverviewGenerator.scala

```diff
@@ -83,8 +83,9 @@ object OverviewGenerator extends LazyLogging {
 
     println("Creating project overview")
     val projectOverviewOption =
-      OverviewGenerator.createProjectOverview(initialProjectScenes,
-                                              overviewInput.pixelSizeMeters)
+      OverviewGenerator.createProjectOverview(
+        initialProjectScenes,
+        156412 / math.pow(2, overviewInput.minZoomLevel))
```

Comment:
Probably a comment explaining where this came from would be helpful

Reply:
@jisantuc is a genius and figured this formula out 💯